stable/sonarqube/templates/pvc.yaml

```diff
@@ -8,6 +8,12 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+{{ if .Values.presistence.annotations}}
```

Comment:
typo here😂

Reply:
haaa :(

Reply:
#15807 
fix here

Reply:
had this error as well.  I thought this was the stable branch.